docker/share/saml/substituter.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/sh
```

Comment:
Should we name this file something slightly more specific?  Maybe something like `saml-config.sh`?

Reply:
Sounds good. Thanks!